openfl/federated/plan/plan.py

```diff
@@ -64,7 +64,8 @@ def ignore_aliases(self, data):
 
     @staticmethod
     def parse(plan_config_path: Path, cols_config_path: Path = None,
-              data_config_path: Path = None, resolve=True):
+              data_config_path: Path = None, resolve=True,
+              working_dir: Path = None):
```

Comment:
It feels like we are polluting the Plan's code with folder management.
We removed two `os.chdir()` lines and then created like 150 more.